Covid19Radar/Covid19Radar/ViewModels/HomePage/HomePageViewModel.cs

```diff
@@ -102,30 +102,50 @@ public override async void Initialize(INavigationParameters parameters)
             // Check Version
             AppUtils.CheckVersion(loggerService);
 
+            await localNotificationService.PrepareAsync();
+
+            await StartExposureNotificationAsync();
+
+            _ = exposureDetectionBackgroundService.ExposureDetectionAsync();
+
+            loggerService.EndMethod();
+        }
+
+        private async Task StartExposureNotificationAsync()
+        {
+            loggerService.StartMethod();
+
             try
             {
-                await localNotificationService.PrepareAsync();
+                var success = await exposureNotificationApiService.StartExposureNotificationAsync();
+                if (success)
```

Comment:
[修正任意]
真偽型は `is` のプリフィクスを付けたいです。

```suggestion
                var isSuccessful = await exposureNotificationApiService.StartExposureNotificationAsync();
                if (isSuccessful)
```

Reply:
eb81000 で対応しました！